Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.17.1
+FROM static-debian11
```

Comment:
Description says distroless but this is not?

Reply:
@mjudeikis what am I missing here? this list says it is 🤔 https://github.com/GoogleContainerTools/distroless

Reply:
We use this same image in other repos BTW

Reply:
You added a full path ;) problem with short path is that it defaults to docker hub and image might not be from the original publisher :)